shared/chat/conversation-filter-input.tsx

```diff
@@ -37,7 +37,7 @@ class ConversationFilterInput extends React.PureComponent<Props> {
     }
   }
 
-  _onEnterKeyDown = (e: React.KeyboardEvent) => {
+  _onEnterKeyDown = (e: React.BaseSyntheticEvent) => {
```

Comment:
in order to share the events between RN and web we go down a level and use the `BaseSyntheticEvent` which has the shared stuff